js/main.js

```diff
@@ -1,17 +1,71 @@
-const comment = document.querySelector('.social__footer-text');
-
-function getRandomNumber(minNumber, maxNumber) {
-  if (maxNumber > minNumber && maxNumber > 0 && minNumber >= 0) {
-    minNumber = Math.ceil(minNumber);
-    maxNumber = Math.floor(maxNumber);
-    return Math.floor(Math.random() * (maxNumber - minNumber + 1)) + minNumber;
-  }
-  const error = 'Введите другие числа';
-  return error;
+function getRandomNumber(firstNumber, secondNumber) {
+  const lower = Math.ceil(Math.min(Math.abs(firstNumber), Math.abs(secondNumber)));
+  const upper = Math.floor(Math.max(Math.abs(firstNumber), Math.abs(secondNumber)));
+  const result = Math.random() * (upper - lower + 1) + lower;
+  return Math.floor(result);
 }
-getRandomNumber(1, 8);
 
 function checkStringLength(nameString, maxValue) {
   return nameString.length <= maxValue;
 }
-checkStringLength(comment, 140);
+checkStringLength('comment', 140);
+
+// Функция нахождения случайного элемента из массива
+function getRandomElementArray (array) {
+  const randomElementIndex = Math.floor(Math.random() * (array.length));
+  return array[randomElementIndex];
+}
+
+// Генерация рандомного неповторяющегося числа
+const numberArray = [];
+const getRandomOriginalNumber = function (array) {
+  const numberIndex = getRandomNumber(1, 25);
```

Comment:
```suggestion
  const numberIndex = getRandomNumber(магические числа);
```

https://up.htmlacademy.ru/javascript/23/criteries#d19